### DISCUSSION TITLE
Suggest adding Rust

### DISCUSSION THREAD

**Discussion by monkeycc** [open-edge-platform/anomalib#1230] [Feature Requests]
High performance

Low resource occupancy

Can be deployed on the desktop WebAssembly embedded development board server

Suggest considering adding Rust  inference Demo

**Comment by samet-akcay** [comment]
@monkeycc, it would be an interesting application. However, we do not have enough resources to work on this. I'm moving this to `Ideas, Feature Requests` section, for open-source contribution.

**Comment by samet-akcay** [comment]
This is something @ashwinvaidya17 is working on his spare time. Once he has some updates, we could share it here.
